packages/amplify-e2e-tests/src/__tests__/geo-update-2.test.ts

```diff
@@ -128,4 +131,35 @@ describe('amplify geo update', () => {
     expect(getGeoJSConfiguration(awsExport).geofenceCollections.default).toEqual(collection2Name);
     expect(getGeoJSConfiguration(awsExport).region).toEqual(region);
   });
+
+  it.only('init a project with default auth config and add two device location tracker resources and update the first to be default', async () => {
```

Comment:
```suggestion
it('init a project with default auth config and add two device location tracker resources and update the first to be default', async () => {
```

Reply:
Also, can you check the runs in CircleCI and ensure that `geo-update-2` is short enough to tack on an additional test? It's less than ideal, but our test runs split based on the file they exist in.

Reply:
Good call out, will try that